waypoint_planner/src/astar_avoid/astar_avoid.cpp

```diff
@@ -370,35 +374,34 @@ tf::Transform AstarAvoid::getTransform(const std::string& from, const std::strin
   return stf;
 }
 
-int AstarAvoid::getLocalClosestWaypoint(const autoware_msgs::Lane& waypoints, const geometry_msgs::Pose& pose, const int& search_size)
+int AstarAvoid::getLocalClosestWaypoint(const autoware_msgs::Lane& waypoints, const geometry_msgs::Pose& pose,
+                                        const int& search_size)
 {
-  static int prev_index = -1;
-  static autoware_msgs::Lane local_waypoints;  // around self-vehicle
-
   // search in all waypoints if lane_select judges you're not on waypoints
   if (closest_waypoint_index_ == -1)
   {
     prev_index = -1;
```

Comment:
Removing line 375 above causes this variable to be undefined when it is used here. This is causing CI failures. Please correct.

Reply:
Thanks for catching! fixed.